### ISSUE TITLE
Purging of unused files

### ISSUE THREAD

**Issue by chintanjosh** [MedicalAppInfibeam/DoctorsWebApp#99]
Purge all the unused files.
You can keep this in a low priority, but don't forget to do this.

**Comment by garvit-gupta** [comment]
@chintanjosh We have removed the migrations folder, the media folder and the .pyc files from the remote repo. Any other files that should be purged?

**Comment by chintanjosh** [comment]
Looks fine as of now, @garvit-gupta do take care while removing migrations folder, initially I said not to include it as there might have been lots of changes so, but once everything is finalized keep it, or else every one will have to run migration command every time. Rest of the things look good.

**Comment by garvit-gupta** [comment]
@chintanjosh Yes sir, after the project is ready we shall add the migration folders too.
